.github/workflows/build.yml

```diff
@@ -38,6 +38,10 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v3
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@fba817f3c0db4f854d7a3ee688241d6754da166e
```

Comment:
can we use a tag rather than pinning?

Reply:
Yes, I've reverted the change to use a commit hash instead of tag now, which goes against AFCMS's suggestion.

Reply:
Ah right, I skipped that discussion.
Pinning is also fine by me.